src/Microsoft.Azure.NotificationHubs/KeyValueConfigurationManager.cs

```diff
@@ -38,28 +54,28 @@ internal class KeyValueConfigurationManager
         private static readonly Regex ValueRegex = new Regex(@"([^\s]+)");
 
         internal NameValueCollection connectionProperties;
-        internal string connectionString;
+        internal string _connectionString;
```

Comment:
If it's internal, we should use `ConnectionString` here instead

Reply:
renamed to ConnectionString